src/_nebari/upgrade.py

```diff
@@ -1182,6 +1182,29 @@ def _version_specific_upgrade(
         return config
 
 
+class Upgrade_2024_7_1(UpgradeStep):
+    """
+    Upgrade step for Nebari version 2024.7.1
+
+    Note:
+        - Digital Ocean deprecation warning.
+    """
+
+    version = "2024.7.1"
+
+    @override
+    def _version_specific_upgrade(
+        self, config, start_version, config_filename: Path, *args, **kwargs
+    ):
+        if config.get("provider", "") == ProviderEnum.gcp.value:
```

Comment:
We want to have `do` instead of `gcp` here

Reply:
ops, my copy and paste failed me

Reply:
done